tools/debian/control

```diff
@@ -70,26 +74,26 @@ Description: Cockpit deployment and developer guide
  or extend Cockpit.
 
 Package: cockpit-docker
-Architecture: amd64 armel armhf i386
+Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python
```

Comment:
See discussion with Peter above. The Debian package does not ship the bits that call Python (allegedly it's only for interacting with Atomic), and I really want to avoid introducing a new Python 2 dependency into Debian/Ubuntu particularly if it's not actually required.